### PR TITLE
Condition added not to show dialogue window in case of UIAutomation.

### PR DESCRIPTION
**Pull request by jinujoseph**
This is to solve #408 
@adrianvmsft

### CODE REVIEW

File: src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/UserNotificationServices.cs [hunk 1]
```diff
@@ -25,22 +25,26 @@ public UserNotificationServices(SVsServiceProvider serviceProvider, IProjectThre
         public bool Confirm(string message)
         {
             _threadingService.VerifyOnUIThread();
-
-            var result = VsShellUtilities.ShowMessageBox(_serviceProvider, message, null, OLEMSGICON.OLEMSGICON_QUERY,
-                         OLEMSGBUTTON.OLEMSGBUTTON_YESNO, OLEMSGDEFBUTTON.OLEMSGDEFBUTTON_FIRST);
-            if (result == (int)VSConstants.MessageBoxResult.IDNO)
+            if (!VsShellUtilities.IsInAutomationFunction(_serviceProvider))
```
Comment (adrianvmsft) [inline src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/UserNotificationServices.cs]: Should it return false when running under automation?

File: src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/UserNotificationServices.cs [hunk 2]
```diff
@@ -25,22 +25,26 @@ public UserNotificationServices(SVsServiceProvider serviceProvider, IProjectThre
...
             }
             return true;
         }
 
         public void NotifyFailure(string failureMessage)
         {
             _threadingService.VerifyOnUIThread();
-
-            var result = VsShellUtilities.ShowMessageBox(_serviceProvider, failureMessage, null, OLEMSGICON.OLEMSGICON_WARNING,
+            if (!VsShellUtilities.IsInAutomationFunction(_serviceProvider))
```
Comment (adrianvmsft) [inline src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/UserNotificationServices.cs]: Should we write something to the log, or at least a trace in this case?

**Comment by davkean** [comment]
👍

**Comment by adrianvmsft** [comment]
👍 
I validated using the private build, the change unblocks our automation.

**Comment by jinujoseph** [comment]
Thanks much Adrian.

**Pull request merged**